week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,25 +19,38 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(sales) {
+  var sales1 = sales * 1.2;
+  console.log("£" + sales1.toFixed(2));
+  return "£" + sales1.toFixed(2);
```

Comment:
while this is a valid solution. The comment above mentioned that you can use the functions you used before. 
something like  (sales1 is a weird name, maybe salesAfterTax)
```var sales1=calculateSalesTax(sales); ```

Reply:
Perfect!